tutorials-and-examples/storage/hyperdisk-ml/README.md

```diff
@@ -0,0 +1,221 @@
+## Populate a Hyperdisk ML Disk from Google Cloud Storage
+
+### This guide uses the Google Cloud API to create a Hyperdisk ML disk from data in Cloud Storage and then use it in a GKE cluster. Refer to [this documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/hyperdisk-ml) for instructions all in the GKE API
+
+1. Create a new GCE instance that you will use to hydrate the new Hyperdisk ML disk with data. Note a c3-standard-44 instance is used to provide the max throughput while populating the hyperdisk([Instance to throughput rates](https://cloud.google.com/compute/docs/disks/hyperdisks#performance_limits_for_other_vms)).
+
+```sh
+VM_NAME=hydrator
+MACHINE_TYPE=c3-standard-44
+IMAGE_FAMILY=debian-11
+IMAGE_PROJECT=debian-cloud
+ZONE=us-central1-a
+SNAP_SHOT_NAME=hdmlsnapshot
+PROJECT_ID=myproject
+DISK_NAME=model1
+
+gcloud compute instances create $VM_NAME \
+    --image-family=$IMAGE_FAMILY \
+    --image-project=$IMAGE_PROJECT \
+    --zone=$ZONE \
+    --machine-type=$MACHINE_TYPE
+
+gcloud compute ssh $VM_NAME
+
+```
+Update and authenticate the instance
+
+```sh
+sudo apt-get update
+sudo apt-get install google-cloud-cli
+gcloud init
+gcloud auth login
+
+```
+
+2. Create and attach the disk to the new GCE VM.
+
+```sh
+SIZE=140
+THROUGHPUT=2400
+
+gcloud compute disks create $DISK_NAME --type=hyperdisk-ml \
+--size=$SIZE --provisioned-throughput=$THROUGHPUT  \
+--zone $ZONE
+
+gcloud compute instances attach-disk $VM_NAME --disk=$DISK_NAME --zone=$ZONE 
+```
+
+3. Log into the hydrator VM, format the volume, initiate transfer, and dismount the volume.
+
+```sh
+gcloud compute ssh $VM_NAME
+```
+
+Mount and load the data from GCS
+```sh 
+lsblk
+```
+
+Save device name given by lsblk(example nvme0n2)
+```sh
+DEVICE=nvme0n2
+```
+
+```sh
+GCS_DIR=gs://vertex-model-garden-public-us-central1/llama2/llama2-70b-hf 
+sudo /sbin/mkfs -t ext4 -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/$DEVICE
+
+sudo mount $DEVICE /mnt
```

Comment:
I think $DEVICE needs to be prefixed with `/dev`. Either udpate the variable above, or prefix this with `/dev/`